docs/source/dashboard.rst

```diff
@@ -95,4 +95,8 @@ Dissecting the dashboard structure
 Searching jobs
 --------------
 
-The search bar accepts JSON-formatted queries in the same way as the ``signac find`` command-line tool. For example, using the query ``{"key": "value"}`` will return all jobs where the job statepoint ``key`` is set to ``value``. To search jobs by their document key-value pairs, use ``doc:`` before the JSON-formatted query, like ``doc:{"key": "value"}``.
+The search bar accepts queries like the ``signac find`` command line tool, including the query namespaces ``'sp.'`` and ``'doc.'``.
+
+Simple queries can be entered readably like **"sp.a 1 doc.b 2"**. This example will return all jobs where the job statepoint ``a`` has the value ``1`` and the job document ``b`` has the value ``2``.
```

Comment:
```suggestion
Simple queries can be entered readably, for example **"sp.a 1 doc.b 2"**. This example will return all jobs where the job statepoint ``a`` has the value ``1`` and the job document ``b`` has the value ``2``.
```

Reply:
I like pointing out it is an eg.
Also trying to remove the word "simple"

```suggestion
Queries checking for values can be entered without the JSON formatting (for example **"sp.a 1 doc.b 2"**). This example will return all jobs where the job statepoint ``a`` has the value ``1`` and the job document ``b`` has the value ``2``.
```